src/components/CreditCard/style.tsx

```diff
@@ -0,0 +1,156 @@
+import styled from 'styled-components'
+import { Colors } from '../../tokens'
+
+const CardContainer = styled.div<{
+  $backgroundCardColor?: string
+  $newCreditCard?: boolean
+}>`
+  max-width: 267px;
+  width: 100%;
+  height: 165px;
```

Comment:
165 is with the shadow. The correct height is 150px.

Reply:
It was fixed, you can see this change on this commit b087ad3790a245ca4bd5ae59ce1af89fc08df75b